src/engine/physics/FindGround.hpp

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <btBulletDynamicsCommon.h>
+
+namespace lei3d 
+{
+    class FindGround : public btCollisionWorld::ContactResultCallback 
+    {
+    public:
+        btRigidBody* m_Me;
+        float m_ShapeHalfHeight = 1.5; // DEFINED IN CHARACTER CONTROLLER COMPONENT 
+        float m_ShapeRadius = 1; // DEFINED IN CHARACTER CONTROLLER COMPONENT 
+        float m_RadiusThreshold = 1e-2;
+        float m_MaxCosGround = -SIMDSQRT12; // approx SQRT(1/2) = sqrt(2)/2 , defined by Bullet3
+        bool m_HaveGround = false;
```

Comment:
What does this mean? Is this when the rigidbody is on the ground? If so prob. want to call it like m_Grounded